ceviche/fdfd.py

```diff
@@ -191,6 +191,123 @@ def _solve_fn(self, eps_vec, entries_a, indices_a, Jz_vec):
         Hx_vec, Hy_vec = self._Ez_to_Hx_Hy(Ez_vec)
         return Hx_vec, Hy_vec, Ez_vec
 
+
+class fdfd_mf_ez(fdfd):
+    """ FDFD class for multifrequency linear Ez polarization """
+
+    def __init__(self, omega, dL, eps_r, Omega, delta, phi, Nsb, npml, bloch_phases=None):
```

Comment:
What is `Omega`?  Perhaps can we change this variable name to be more descriptive and not to get it confused with `omega` (undercase)?

Reply:
Also, maybe we should add some documentation for each of these arguments (docstring) so that people know what they are and what datastructures to use for them.  A simple docstring will work, no preference.

Reply:
Changed `Omega` to `omega_mod`, added documentation.